test/unit/repos/pseudonyms.test.js

```diff
@@ -44,6 +45,55 @@ describe('Pseudonyms repo', () => {
     jest.clearAllMocks()
   })
 
+  describe('sortBy', () => {
+    const list = [
+      {
+        rowKey: 1,
+        timestamp: '2024-04-11T08:00:43.8624424Z',
+        username: 'Joey_Flatley'
+      },
+      {
+        rowKey: 2,
+        timestamp: '2024-04-11T08:01:24.8314117Z',
+        username: 'Jeremy83'
+      },
+      {
+        rowKey: 3,
+        timestamp: '2024-04-11T08:01:24.8664459Z',
+        username: 'Kendrick_Wuckert'
+      },
+      {
+        rowKey: 4,
+        timestamp: '2024-04-11T08:01:24.8904697Z',
+        username: 'Shane63'
+      }
+    ]
+
+    describe('sortByTimestamp', () => {
```

Comment:
The only thing I'd suggest is having your timestamps in non-linear order in the test data. Currently, if no sorting occurs, your 'ascending' test passes

Reply:
good spot, thanks